tests/integration/api_v3/http/posStakers.test.js

```diff
@@ -49,6 +49,20 @@ describe('Stakers API', () => {
 			expect(response.data.stakers.length).toBeLessThanOrEqual(10);
 		});
 
+		it('Returns list of stakers when requested for known validator address and search param (exact staker name)', async () => {
+			const response = await api.get(`${endpoint}?address=${refValidator.address}&search=${refValidator.name}`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data.stakers.length).toBe(1);
```

Comment:
Kindly also assert that the returned `stakerAddress` is an exact match for `refValidator.address`.

Reply:
Applied.